smarts/core/sensors.py

```diff
@@ -448,7 +448,7 @@ def _is_done_with_events(cls, sim, agent_id, vehicle, sensor_state):
         collided = sim.vehicle_did_collide(vehicle.id)
         is_off_road = cls._vehicle_is_off_road(sim, vehicle)
         is_on_shoulder = cls._vehicle_is_on_shoulder(sim, vehicle)
-        is_not_moving = cls._vehicle_is_not_moving(sim, vehicle)
+        is_not_moving = cls._vehicle_is_not_moving(sim, vehicle, done_criteria.not_moving[1])
```

Comment:
My point here (see my previous comment). This result is has implications on the `Events` object as well as `done`.